spec/duration.html

```diff
@@ -1729,79 +1728,82 @@ <h1>
           1. Assert: _fractionalDays_ is not used below.
```

Comment:
This assertion is confusing. How far below?  Because it is indeed used down in line 1742. Maybe there's another way to say it that's less confusing?

Reply:
This is unrelated to this PR so I'd prefer to address it later.

FWIW, the full context is

- If _unit_ is **"year"**, **"month"**, **"week"**, or **"day"**, then
   - Let _fractionalDays_ be some value.
   - Assert: _fractionalSeconds_ is not used below.
 - Else,
   - Let _fractionalSeconds_ be some value.
   - Assert: _fractionalDays_ is not used below.

The idea is that each of the calculations for each `smallestUnit` uses either the value in _fractionalDays_ or _fractionalSeconds_, but not both.

Any suggestions on how to express this better?
Maybe `Set _fractionalDays_ to ~unused~` so it's intentionally an editorial error if you try to do arithmetic with it?

Reply:
After looking at the spec text outside of the PR diff, I think it's fine as-is. In the diff here the nesting and context isn't as obvious as in an editor, where it's clear what you mean. Resolving.